submissions/KostiantynOkhorzin/dom/script.js

```diff
@@ -0,0 +1,176 @@
+            
+            HBO ordered the seventh season on April 21, 2016, three days before the premiere of the show's sixth season and began filming on August 31, 2016. The season was filmed primarily in Northern Ireland, Spain, Croatia and Iceland. Game of Thrones features a large ensemble cast, including Peter Dinklage, Nikolaj Coster-Waldau, Lena Headey, Emilia Clarke and Kit Harington. The season introduces several new cast members, including Jim Broadbent and Tom Hopper. The series received 22 nominations for the 70th Primetime Emmy Awards and won for Outstanding Drama Series and Dinklage won for Outstanding Supporting Actor in a Drama Series.`,
+            img: 'img/season7.jpg'
+        },
+        {
+            title: 'Game of Thrones (season 8)',
+            descr: `The eighth and final season of the fantasy drama television series Game of Thrones, produced by HBO, premiered on April 14, 2019, and concluded on May 19, 2019. Unlike the first six seasons, which consisted of ten episodes each, and the seventh season, which consisted of seven episodes, the eighth season consists of only six episodes.
+
+            The season was filmed from October 2017 to July 2018 and largely consists of original content not found in George R. R. Martin's A Song of Ice and Fire series, while also incorporating material that Martin has revealed to showrunners about the upcoming novels in the series, The Winds of Winter and A Dream of Spring. The season was adapted for television by David Benioff and D. B. Weiss.
+            
+            The season was met with mixed reviews from critics, in contrast to the critical acclaim of previous seasons, and is the lowest-rated of the series on the website Rotten Tomatoes. While the performances, production values and music score were praised, criticism was mainly directed at the shorter runtime of the season as well as numerous creative decisions made by the showrunners regarding the plot and character arcs; many commentators deemed it to be a disappointing conclusion to the series.
+            
+            The season received 32 nominations at the 71st Primetime Emmy Awards, the most for a single season of television in history. It won twelve, including Outstanding Drama Series and Outstanding Supporting Actor in a Drama Series for Peter Dinklage.`,
+            img: 'img/season8.jpg'
+        },
+    ];
+
+    const listBtns = document.querySelector('.side-menu');
+    const content = document.querySelector('.content');
+    const burger = document.querySelector('.burger');
+
+
+    const renderBtns = (arr, list) => {
+        arr.map((item, i) => {
```

Comment:
Why do you use this method? The variable `item` is not used. Set a more informative name for variables.

Reply:
You misunderstood me a little
The method `Array.map()` iterates over the values, makes some changes and returns a new array. In this case, I would recommended the method `Array.forEach()`.

Reply:
Should I rewrite on the forEach or leave for?

Reply:
Yes, it is better to use the method `forEach`

Reply:
But then the variable item is not used.